README.md

```diff
@@ -17,6 +17,29 @@
 ### Installation
 To install this extension, download the latest release of Redirect and drag the `redirect.blueprint` file over to your Pterodactyl installation folder. Then, run `blueprint -install redirect` to finish the installation process.
 
+If you're using NGINX as your webserver, assuming you followed the Pterodactyl installation instructions, you will also need to modify your config file. Apache and Caddy should work without additional configuration.
+
+#### NGINX Configuration
+- Open your NGINX server config file, typically located in `/etc/nginx/sites-available/`
+- Within your `server` block that contains the `index` line (the second server block, if you followed the Pterodactyl guide), change `index index.php` to `index index.php index.html`
+- Save the file, then test your config with `nginx -t`
+- If the test completes without issue, restart NGINX with `sudo systemctl restart nginx`
```

Comment:
Could be modified slightly to expand on ways to restart NGINX on other flavors of OS, but I think the average technical knowledge required to get to this point should allow this simplification to be fine